src/timepicker/timepicker.ts

```diff
@@ -81,51 +86,99 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
             [readonly]="readonlyInputs" [disabled]="disabled" aria-label="Hours" i18n-aria-label="@@ngb.timepicker.hours">
-          <button *ngIf="spinners" type="button" class="btn btn-link" [ngClass]="setButtonSize()" (click)="changeHour(-hourStep)"
-            [disabled]="disabled" [class.disabled]="disabled">
+          <button
+            *ngIf="spinners"
+            tabindex="-1"
+            type="button"
+            class="btn btn-link"
+            [ngClass]="setButtonSize()"
+            (click)="changeHour(-hourStep)"
+            [disabled]="disabled"
+            [class.disabled]="disabled"
+          >
             <span class="chevron bottom"></span>
             <span class="sr-only" i18n="@@ngb.timepicker.decrement-hours">Decrement hours</span>
           </button>
         </div>
         <div class="ngb-tp-spacer">:</div>
         <div class="ngb-tp-input-container ngb-tp-minute">
-          <button *ngIf="spinners" type="button" class="btn btn-link" [ngClass]="setButtonSize()" (click)="changeMinute(minuteStep)"
-            [disabled]="disabled" [class.disabled]="disabled">
+          <button
+            *ngIf="spinners"
+            tabindex="-1"
+            type="button"
+            class="btn btn-link"
+            [ngClass]="setButtonSize()"
+            (click)="changeMinute(minuteStep)"
+            [disabled]="disabled"
+            [class.disabled]="disabled"
+          >
             <span class="chevron"></span>
             <span class="sr-only" i18n="@@ngb.timepicker.increment-minutes">Increment minutes</span>
           </button>
           <input type="text" class="form-control" [ngClass]="setFormControlSize()" maxlength="2"
             placeholder="MM" i18n-placeholder="@@ngb.timepicker.MM"
-            [value]="formatMinSec(model?.minute)" (change)="updateMinute($event.target.value)"
+            [value]="formatMinSec(model?.minute)" (keydown)="handleMinuteKeyDown($event)" (change)="updateMinute($event.target.value)"
             [readonly]="readonlyInputs" [disabled]="disabled" aria-label="Minutes" i18n-aria-label="@@ngb.timepicker.minutes">
-          <button *ngIf="spinners" type="button" class="btn btn-link" [ngClass]="setButtonSize()" (click)="changeMinute(-minuteStep)"
-            [disabled]="disabled" [class.disabled]="disabled">
+          <button
+            *ngIf="spinners"
+            tabindex="-1"
+            type="button"
+            class="btn btn-link"
+            [ngClass]="setButtonSize()"
+            (click)="changeMinute(-minuteStep)"
+            [disabled]="disabled"
+            [class.disabled]="disabled"
+          >
             <span class="chevron bottom"></span>
             <span class="sr-only"  i18n="@@ngb.timepicker.decrement-minutes">Decrement minutes</span>
           </button>
         </div>
-        <div *ngIf="seconds" class="ngb-tp-spacer">:</div>
+        <div *ngIf="seconds" class="ngb-tp-spacer">:</div>=
```

Comment:
You have added the `=` symbol in the template

![screen shot 2018-06-28 at 17 59 51](https://user-images.githubusercontent.com/8074436/42045966-19bc22be-7afd-11e8-9fcf-137246694805.png)